### PR TITLE
Run 'docker version' in dockerSetup.

### PR DESCRIPTION
**Pull request by jkozlowski**
- Should print something like the following if docker is down:

```
:dockerSetup
    Client:
    Cannot connect to the Docker daemon. Is the docker daemon running on this host?
     Version:      1.12.0-rc2
     API version:  1.24
     Go version:   go1.6.2
     Git commit:   906eacd
     Built:        Fri Jun 17 20:35:33 2016
     OS/Arch:      darwin/amd64
     Experimental: true
    :dockerSetup FAILED
```

Should fix #37. Looking for ideas on how to test this; I tried sudo service stop docker, but I'm not sure it is actually possible to stop the docker daemon on CircleCI?

### CODE REVIEW

**Comment by jkozlowski** [comment]
Or just remove it altogether, the output is fairly informative from just running the docker commands.

File: src/main/groovy/com/palantir/gradle/docker/DockerSetupTask.groovy [hunk 1]
```diff
@@ -30,32 +28,14 @@ class DockerSetupTask extends DefaultTask {
...
-            if (!System.env.DOCKER_CERT_PATH) {
-                error += 'DOCKER_CERT_PATH not set.\n'
-            }
-            if (!System.env.DOCKER_MACHINE_NAME) {
-                error += 'DOCKER_MACHINE_NAME not set.\n'
-            }
-            if (error != '') {
-                throw new GradleException(error += 'Please make sure your Docker VM is running.')
-            }
+        project.exec {
```
Comment (uschi2000) [inline src/main/groovy/com/palantir/gradle/docker/DockerSetupTask.groovy:36]: does this fail with a reasonable error?

Comment (jkozlowski) [inline src/main/groovy/com/palantir/gradle/docker/DockerSetupTask.groovy:36]: You get what I pasted in the description and some version information: 

```
Cannot connect to the Docker daemon. Is the docker daemon running on this host?
```

Can change to docker ps, the output will be less noisy. Or in general get rid of this task, all the other ones will fail with the same message when cannot connect.

Comment (markelliot) [inline src/main/groovy/com/palantir/gradle/docker/DockerSetupTask.groovy:36]: I think just kill the task entirely.

Comment (jkozlowski) [inline src/main/groovy/com/palantir/gradle/docker/DockerSetupTask.groovy:36]: here you go: #56

**Comment by markelliot** [comment]
Took #56 as preferred alternative.

**Pull request closed without merging**